astro/src/components/BandInfo.astro

```diff
@@ -120,6 +121,5 @@ const builder = sanityImageUrl(groupPhoto.image)
   }
 
   animateOnScroll()
-
-  // fadeIn();
+  document.addEventListener("astro:page-load", animateOnScroll)
```

Comment:
With the addition of client-side routing to the app, a lot of this vanilla JS animation wasn't working. Needed to add a listener for the `astro:page-load` event that fires after successful client-side navigation.

[More context](https://docs.astro.build/en/guides/view-transitions/#script-behavior-during-page-navigation)